opm/simulators/linalg/ParallelIstlInformation.hpp

```diff
@@ -257,21 +248,22 @@ class ParallelISTLInformation
             }
             else
             {
-                return getInitialValue();
+                return getInitialValue<T>();
             }
         }
-        Result getInitialValue()
+        template<class T>
+        T getInitialValue()
         {
             //g++-4.4 does not support std::numeric_limits<T>::lowest();
             // we rely on IEE 754 for floating point values and use min()
             // for integral types.
-            if( std::is_integral<Result>::value )
+            if( std::is_integral<T>::value )
```

Comment:
As we are touching this anyway and are beyond g++4.4, maybe std::numeric_limits<T>::lowest() would work now?

Reply:
I don't see why not. It's done.